File: CSharp/HtmlWalker/Exception/TagDeprecatedException.cs
using HtmlWalker.Core;

namespace HtmlWalker.Exception
{
    public class TagDeprecatedException : HtmlWalkerException
    {
        public TagDeprecatedException(string msg) : base(msg) { }

        public TagDeprecatedException(ITag adapter) : base(adapter.Name) { }
    }
}

File: Java/htmlwalker/src/main/java/htmlwalker/exception/TagDeprecatedException.java
package htmlwalker.exception;

import htmlwalker.ITag;

@SuppressWarnings("serial")
public class TagDeprecatedException extends HtmlWalkerException
{
	
    public TagDeprecatedException(String msg) { super(msg); }

    public TagDeprecatedException(ITag adapter) { super(adapter.name()); }
    
}
